tools/dna_rna_tools.py

```diff
@@ -0,0 +1,75 @@
+def valid(seq):  # ф-ия проверки существования посл-ти
+    for d in seq[: -1]:
+        if ('U' in d or 'u' in d) and ('T' in d or 't' in d):
+            return False
+
+
+def transcribe(seq):  # ф-ия транскрибирования
+    trans = []
+    for i in seq:
```

Comment:
```suggestion
    for nucl in seq:
```

писал про это в прошлый раз

-1